includes/class-wc-stripe-update-manager.php

```
@@ -0,0 +1,92 @@
+	 *
+	 * @return callable[]
+	 */
+	protected function get_update_functions(): array {
+		$functions = [];
+
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/admin/class-wc-stripe-admin-notices.php';
+
+		$functions[] = [ WC_Stripe_Admin_Notices::class, 'check_update_notices' ];
+
+		require_once WC_STRIPE_PLUGIN_PATH . '/includes/migrations/class-allowed-payment-request-button-types-update.php';
+		$functions[] = [ new Allowed_Payment_Request_Button_Types_Update(), 'maybe_migrate' ];
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '82', column '24'). (undefined)

(MissingImport)